test/local/account_routes.js

```diff
@@ -1531,16 +1531,17 @@ test('/recovery_email/verify_code', function (t) {
   var mockDB = mocks.mockDB(dbData, dbErrors)
   var mockLog = mocks.spyLog()
   var mockMailer = mocks.mockMailer()
+  var mockCustoms = {
```

Comment:
Worthwhile to pull this into [mock.js](https://github.com/mozilla/fxa-auth-server/blob/master/test/mocks.js)?